modules/javafx.graphics/src/main/java/javafx/css/TransitionEvent.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package javafx.css;
+
+import javafx.animation.Transition;
+import javafx.event.Event;
+import javafx.event.EventType;
+import javafx.scene.Node;
+import javafx.util.Duration;
+
+/**
+ * An event that signals the creation, beginning, completion and cancellation of implicit CSS transitions.
+ * <p>
+ * Note that this event is not raised for explicit transitions that are created using the {@link Transition} class.
+ *
+ * @since 22
+ */
+public final class TransitionEvent extends Event {
+
+    private static final long serialVersionUID = 20220820L;
```

Comment:
minor: may I suggest `1L` as in version 1?  It's not an existing class that needs to be compatible with older implementations for which a version was derived by the compiler.